src/temp.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+</head>
+<body>
+<header>
+    <div class="link before">
+        <a href="#" class="nav-link indent active"><h1>PORTFOLIO</h1></a>
```

Comment:
It is not correct position of H1! "Portfolio" is a menu item, but no a real header. Remove it from here and think about h1.